ido/tests/sim_test/ido_test/whitelist_test.rs

```diff
@@ -0,0 +1,217 @@
+use crate::utils::init;
+use near_sdk::serde_json::json;
+use near_sdk::json_types::{U64, U128};
+use near_sdk_sim::{DEFAULT_GAS, to_yocto};
+use kulapad_ido::modules::project::{ProjectJson,ProjectStatus,WhitelistType};
+use kulapad_ido::modules::account::ProjectAccountJson;
+use kulapad_ido::staking_contract::AccountJson;
+
+#[test]
+pub fn test_join_whitelist(){
+    let (root,alice,ido,_,_,ido_contract) = init();
+    
+    ido.call(
+        ido_contract.account_id(),
+        "update_project_whitelist_date",
```

Comment:
The `update_project_whitelist_date` is helpful when we do the demo. For the simulation tests, we should change the `block_timestamp` instead.